app/assets/stylesheets/templates/_messages.scss

```diff
@@ -2,8 +2,95 @@
   .slab--header {
     padding: {
       top: 1em;
-      bottom: 1em;
+      bottom: .5em;
+    }
+    h2, p {
+    	margin-bottom: 0;
     }
   }
-
 }
+.message--container {
+	display: flex;
```

Comment:
style nitpick: For consistency with the rest of the file/project you should configure your text editor to use two-space indents rather than a hard tab. (Happy to help you do this if you would like!)